samples/start-app/rust-actix/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+name = "rust-actix"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+path = "src/lib.rs"
+
+[[bin]]
+path = "src/main.rs"
+name = "sample-app"
+
+
+[dependencies]
+tokio = { version = "1.32", features = ["macros", "rt-multi-thread"] }
+async-trait = "0.1"
+actix-web = { version = "4.4", features = ["secure-cookies"] }
+actix-cors = "0.6"
+actix-session = { version = "0.7", features = ["cookie-session"] }
+sailfish = { version = "0.8", features = ["derive"] }
+jsonwebtoken = "8"
+reqwest = { version = "0.11", features = ["blocking", "json"] }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+r2d2 = "0.8"
+r2d2_sqlite = "0.22"
+rusqlite = { version = "0.29", features = ["bundled"] }
+url = { version="2.4", features = ["serde"] }
+rand = "0.8"
+sha2 = "0.10"
+hex = "0.4"
+regex = "1.9"
+anyhow = "1.0"
+dotenv = "0.15"
+env_logger = "0.10"
+tracing-actix-web = "0.7"
+tracing = "0.1"
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+log = "0.4"
+tracing-log = "0.1"
+tracing-bunyan-formatter = "0.3"
+
+[dev-dependencies]
+reqwest = "0.11"
```

Comment:
both in dev-deps and deps?

Reply:
Oh, yes, it's removed on "dev-dependencies".